packages/@aws-cdk/cloud-assembly-schema/lib/manifest.ts

```diff
@@ -42,21 +64,21 @@ export class Manifest {
     return require('../schema/cloud-assembly.version.json').version;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-require-imports
-  private static schema: jsonschema.Schema = require('../schema/cloud-assembly.schema.json');
-
-  private static validate(manifest: assembly.AssemblyManifest) {
+  /**
+   * @internal
+   */
+  public static _validate(assmbly: assembly.Assembly, version: string) {
```

Comment:
Why is this public?

Reply:
It's internal, I think accessed from unit tests, but let me double-check

Reply:
Fixed.